public/main.css

```diff
@@ -0,0 +1,41 @@
+/* line 2, ../src/stylesheets/globals/_reset.scss */
```

Comment:
I had to `sudo gem install sass` -- but my main.css != to yours:
```
mmccorkl-mac-lt:SEWebFE-erindepew mmccorkle$ sass src/stylesheets/main.scss ~/Desktop/main.css
mmccorkl-mac-lt:SEWebFE-erindepew mmccorkle$ md5 ~/Desktop/main.css ./public/main.css
MD5 (/Users/mmccorkle/Desktop/main.css) = 7c662844309bda32acfb144529f0105f
MD5 (./public/main.css) = 9fab89f63ae52a2e20944e7d2c861833
mmccorkl-mac-lt:SEWebFE-erindepew mmccorkle$ wc -l ~/Desktop/main.css ./public/main.css
      27 /Users/mmccorkle/Desktop/main.css
      41 ./public/main.css
      68 total
```

What am I doing wrong in my compiling of the sass to css?